File: Linguagem C Exercicios/C-Sequencial/MEDIDAS.c
#include <stdio.h>
#include <stdlib.h>

int main()
{

    double a,b,c,areaTriangulo,areaQuadrado,areaTrapezio = 0;

    printf("Digite o valor de A: ");
    scanf("%lf",&a);
    printf("Digite o valor de B: ");
    scanf("%lf",&b);
    printf("Digite o valor de C: ");
    scanf("%lf",&c);

    areaQuadrado = a*a;
    areaTriangulo = (a*b) / 2.00;
    areaTrapezio = ((a+b)*c) / 2.00;

    printf("Area do Quadrado: %.4lf\n",areaQuadrado);
    printf("Area do Triangulo: %.4lf\n",areaTriangulo);
    printf("Area do Trapezio: %.4lf",areaTrapezio);

    return 0;
}

File: Linguagem C#/Sequencial/Medidas.cs
using System;
using System.Globalization;

namespace Medidas {
    class Program {
        static void Main(string[] args) {

            CultureInfo CI = CultureInfo.InvariantCulture;

            double a, b, c, areaQuadrado, areaTriangulo, areaTrapezio;

            Console.Write("Digite a medida A: ");
            a = double.Parse(Console.ReadLine(),CI);
            Console.Write("Digite a medida B: ");
            b = double.Parse(Console.ReadLine(), CI);
            Console.Write("Digite a medida C: ");
            c = double.Parse(Console.ReadLine(), CI);

            areaQuadrado = a * a;
            areaTriangulo = (a * b) / 2;
            areaTrapezio = ((a + b) * c) / 2;

            Console.WriteLine("AREA DO QUADRADO = "+areaQuadrado.ToString("F4",CI));
            Console.WriteLine("AREA DO TRIANGULO = "+areaTriangulo.ToString("F4",CI));
            Console.WriteLine("AREA DO TRAPEZIO = "+areaTrapezio.ToString("F4",CI));
        }
    }
}
